config.go

```diff
@@ -16,13 +16,15 @@ type Config struct {
 	PowerPin  string `toml:"power-pin"`
 	OutputDir string `toml:"output-dir"`
 	MinSecs   int    `toml:"min-secs"`
+	MaxLen    int    `toml:"max-len"`
 }
 
 var defaultConfig = Config{
 	SPISpeed:  30000000,
 	PowerPin:  "GPIO23",
 	OutputDir: ".",
 	MinSecs:   10,
+	MaxLen:    5400,
```

Comment:
The option should be in seconds and thermal-recorder should convert it to frames when it uses it (using the `framesHz` const that's already there)